admin/tests/integration/components/user-detail-personal-information/user-overview-test.js

```diff
@@ -0,0 +1,285 @@
+      });
+    });
+
+    test('should display the edit and cancel buttons', async function (assert) {
+      // given
+      this.set('user', {
+        firstName: 'John',
+        lastName: 'Harry',
+        email: 'john.harry@example.net',
+        username: null,
+      });
+
+      // when
+      await render(hbs`<UserDetailPersonalInformation::UserOverview @user={{this.user}} />`);
+      await clickByLabel('Modifier');
+
+      // then
+      assert.contains('Editer');
+      assert.contains('Annuler');
+    });
+
+    test('should display user’s first name and last name in edit mode', async function (assert) {
+      // given
+      this.set('user', user);
+
+      // when
+      await render(hbs`<UserDetailPersonalInformation::UserOverview @user={{this.user}}/>`);
+      await clickByLabel('Modifier');
+
+      // then
+      assert.dom('.user-edit-form__first-name').hasValue(this.user.firstName);
+      assert.dom('.user-edit-form__last-name').hasValue(this.user.lastName);
+    });
+
+    module('when user has an email only', function () {
+      test('should display user’s email in edit mode', async function (assert) {
+        // given
+        this.set('user', user);
+
+        // when
+        await render(hbs`<UserDetailPersonalInformation::UserOverview @user={{this.user}}/>`);
+        await clickByLabel('Modifier');
+
+        // then
+        assert.dom('.user-edit-form__email').hasValue(this.user.email);
+      });
+
+      test('should not display username in edit mode', async function (assert) {
+        // given
+        this.set('user', user);
+
+        // when
+        await render(hbs`<UserDetailPersonalInformation::UserOverview @user={{this.user}}/>`);
+        await clickByLabel('Modifier');
+
+        // then
+        assert.notContains('Identifiant :');
+      });
+    });
+
+    module('when user has a username only', function () {
+      test('should display user’s username in edit mode', async function (assert) {
+        // given
+        const user = EmberObject.create({
+          lastName: 'Harry',
+          firstName: 'John',
+          email: null,
+          username: 'user.name1212',
+        });
+        this.set('user', user);
+
+        // when
+        await render(hbs`<UserDetailPersonalInformation::UserOverview @user={{this.user}} />`);
+        await clickByLabel('Modifier');
+
+        // then
+        assert.dom('.user-edit-form__username').hasValue(this.user.username);
+      });
+
+      test('should display email', async function (assert) {
+        // given
+        const user = EmberObject.create({
+          lastName: 'Harry',
+          firstName: 'John',
+          email: null,
+          username: 'user.name1212',
+        });
+        this.set('user', user);
+
+        // when
+        await render(hbs`<UserDetailPersonalInformation::UserOverview @user={{this.user}} />`);
+        await clickByLabel('Modifier');
+
+        // then
+        assert.contains('E-mail :');
```

Comment:
Si chipotage pris en compte 
```suggestion
        assert.contains('Adresse e-mail :');
```